### PR TITLE
f-footer@4.24.1 - Fix failing tests

### PR DESCRIPTION
**Pull request by xander-marjoram**
### Fixed
- Update expected URL for BE and LU

### Changed
- Use domain names in country selector tests

---

## UI Review Checks

- [ ] README and/or UI Documentation has been [created|updated]
- [x] Tests have been [created|updated]
- [ ] This code has been checked with regard to [our accessibility standards](http://fozzie.just-eat.com/documentation/general/accessibility/checklist)

## Browsers Tested

- [ ] Chrome (latest)
- [ ] Internet Explorer 11
- [ ] Mobile (Please list device/browser – Ideally one iPhone model and one Android)

### List any other browsers that this PR has been tested in:

- [View the Browser Support Checklist](http://fozzie.just-eat.com/documentation/general/browser-support)

### CODE REVIEW

**Review by siggerzz — DISMISSED** [review]

**Review by matthewhardern — DISMISSED** [review]

**Review by matthewhardern — DISMISSED** [review]

**Review by siggerzz — DISMISSED** [review]

**Review by matthewhardern — DISMISSED** [review]

File: packages/components/organisms/f-footer/test/component/f-footer.component.desktop.spec.js [hunk 1]
```diff
@@ -173,16 +183,7 @@ describe('Desktop - f-footer component tests - @browserstack', () => {
...
-        ['nz', '.nz'],
-        ['pl', '.pl'],
-        ['pt', '/pt'],
-        ['ro', '/ro'],
-        ['es', '.es'],
-        ['ch_ch', '.ch'],
-        ['ch_en', '/en'],
-        ['ch_fr', '/fr']])
+    ])
     .it('should display link for country code "%s" and redirect to correct URL ("%s")', (country, expectedUrl) => {
```
Comment (xander-marjoram) [inline packages/components/organisms/f-footer/test/component/f-footer.component.desktop.spec.js]: These two tests were identical so I merged them

File: packages/components/organisms/f-header/test/component/f-header.component.desktop.spec.js [hunk 2]
```diff
@@ -84,45 +84,31 @@ describe('Desktop - f-header component tests - @browserstack', () => {
...
+        ['no', 'just-eat.no'],
+        ['pl', 'pyszne.pl'],
+        ['pt', 'takeaway.com/pt'],
+        ['ro', 'takeaway.com/ro'],
+        ['es', 'just-eat.es'],
+        ['ch_ch', 'eat.ch'],
+        ['ch_en', 'eat.ch/en'],
+        ['ch_fr', 'eat.ch/fr']
+    ])
     .it('should display link for country code "%s" and redirect to correct URL ("%s")', (expectedLocale, expectedUrl) => {
```
Comment (xander-marjoram) [inline packages/components/organisms/f-header/test/component/f-header.component.desktop.spec.js:112]: These two tests were identical so I merged them

File: packages/components/organisms/f-footer/test/component/f-footer.component.desktop.spec.js (hunk 1 above)
Comment (dandel10n) [reply packages/components/organisms/f-footer/test/component/f-footer.component.desktop.spec.js:187]: I think you forgot to remove lines 170-185 here

Comment (dandel10n) [reply packages/components/organisms/f-footer/test/component/f-footer.component.desktop.spec.js]: You still have empty `forEach([])` left with the duplicated act and assert sections

Comment (xander-marjoram) [reply packages/components/organisms/f-footer/test/component/f-footer.component.desktop.spec.js]: Ah whoops

**Review by oliversweb — DISMISSED** [review]

**Review by dandel10n — APPROVED** [review]

**Review by oliversweb — APPROVED** [review]

**Pull request merged**